modules/n1ql/pages/n1ql-language-reference/join.adoc

```diff
@@ -248,40 +246,39 @@ NOTE: The LEFT OUTER JOIN will list all left-side results regardless of matching
 
 [source,N1QL]
 ----
-SELECT DISTINCT  MIN(aport.airportname) AS Airport_Name,
-                 MIN(lmark.name) AS Landmark_Name,
-                 MIN(aport.tz) AS Landmark_Time
-FROM `travel-sample` aport
-RIGHT JOIN `travel-sample` lmark
+SELECT DISTINCT MIN(aport.airportname) AS Airport__Name,
```

Comment:
What's this double-underscore convention?

Reply:
I wondered this as well, I think it's just so that all the result field names have the same length and the values are aligned. 🤷‍♂️ left it as-is